keyboards/planck/keymaps/smittey/rules.mk

```diff
@@ -0,0 +1,3 @@
+SRC += muse.c
+BOOTLOADER = atmel-dfu
```

Comment:
```suggestion
```

This shouldn't be necessary unless you're using rev1 or rev2, and rev5 uses qmk-dfu. rev6 has an ARM chip which does not need `BOOTLOADER` at all.